src/DatePicker/DatePicker.test.js

```diff
@@ -101,10 +134,34 @@ describe('<DatePicker />', () => {
         expect(wrapper.state('formattedDate')).toEqual(switchFormattedDate);
         expect(wrapper.state('arrSelectedDates').length).toEqual(2);
 
+        // click on body element
+        let event = new MouseEvent('mousedown', {
+            target: document.querySelector('body')
+        });
         document.dispatchEvent(event);
```

Comment:
I don't understand why the added `mousedown` event, but then no additional assertions.  I'm probably just missing something.

Reply:
added assertion to check that calendar is hidden